.github/workflows/publish-version.yml

```diff
@@ -643,13 +643,13 @@ jobs:
           set -x
 
           # Get the date of the last commit
-          date=$(git show --no-patch --format=%ad --date=format:%Y%m%d)
+          date=$(git show --no-patch --format=%ad --date=format:%Y%m%d%H%M%S)
 
           # Derive crate version
           currentVersion=$(cargo metadata --format-version 1 --no-deps | jq -r '.packages | map(select(.name == "surrealdb"))[0].version')
           major=$(echo $currentVersion | tr "." "\n" | sed -n 1p)
           minor=$(echo $currentVersion | tr "." "\n" | sed -n 2p)
-          # This sets the nightly version to something like `1.3.20231117`
+          # This sets the nightly version to something like `1.3.20250224221932`
```

Comment:
May be minute granularity is enough %H%M and will make it a bit shorter? `1.3.202502242219`